libshaderc_spvc/src/spvcir_pass.cc

```diff
@@ -23,25 +23,46 @@ SpvcIrPass::SpvcIrPass(spirv_cross::ParsedIR *ir) {
   ir_ = ir;
   current_function_ = nullptr;
   current_block_ = nullptr;
+  status_ = Status::SuccessWithoutChange;
 
   auto s = ir_->spirv.data();
```

Comment:
I preferential to not doing any work/processing in the constructor, I would move this to beginning of Process().